app/ui/mainwindow2.ui

```diff
@@ -981,6 +982,11 @@
     <string>Peg bar Alignment</string>
    </property>
   </action>
+  <action name="actionImportLayers_from_pclx">
+   <property name="text">
+    <string>Layers from *.pclx</string>
```

Comment:
The import name doesn't have to be this explicit, I think we could call it "PCLX" without anything else and that would be fine.
A project file consists of layers and frames, so there's no need to mention that explicitly.

Reply:
Just to be sure: you want me to change 'actionImportLayers_from_pclx' to 'actionImport_PCLX'?

Reply:
Naah the action itself is fine I was talking about the UI string that is shown 
eg. "Layers from *.pclx" -> "pclx"

Reply:
I could do that but I think it's too uninformative. Wouldn't it give the impression that you import the whole pclx file, without being able to pick from it?

Reply:
Well you can technically import the whole project file so I don't think that's wrong but in that case it could be written as "Layers from pclx" The extension is imo. unnecessary.